frontend/packages/container-security/src/components/ImageVulnerabilityToggleGroup.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { ChartDonut } from '@patternfly/react-charts';
+import { SecurityIcon } from '@patternfly/react-icons';
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+import { MsgBox } from '@console/internal/components/utils';
+import { totalVulnFor, vulnPriority } from '../const';
+import { ImageManifestVuln, Vulnerability } from '../types';
+import {
+  getFixableVulnerabilitiesCount,
+  getVulnerabilityType,
+  VulnerabilitiesType,
+} from './image-vulnerability-utils';
+import './ImageVulnerabilityTogggleGroup.scss';
+
+type ImageVulnerabilityToggleGroupProps = {
+  obj: ImageManifestVuln;
+};
+
+const ImageVulnerabilityToggleGroup: React.FC<ImageVulnerabilityToggleGroupProps> = ({ obj }) => {
+  const { t } = useTranslation();
+  const vulnerabilities: Vulnerability[] = _.flatten(
+    obj?.spec.features.map((feature) =>
+      feature.vulnerabilities.map((vulnerability) => vulnerability),
+    ),
```

Comment:
you can destructure the obj and use `useMemo` for this. I'm not getting the inner map. this would be same.. right?

```suggestion
  const vulnerabilities: Vulnerability[] = _.flatten(
    obj?.spec.features.map((feature) => feature.vulnerabilities),
```